src/main/java/com/btkelly/gnag/models/ViolationComment.java

```diff
@@ -25,7 +25,7 @@
 
     public ViolationComment(boolean failBuild, String commentMessage) {
```

Comment:
I've actually tried to add these but I always get compilation errors. I didn't spend much time, or any for that matter, trying to figure out why the annotations were not working.